docs/serving/autoscaling/README.md

```diff
@@ -0,0 +1,20 @@
+Knative Serving provides users with the functionality to automatically scale the number of application replicas based on demand.
+
+The component responsible for autoscaling functionality in Knative Serving is the **Autoscaler** component. The Autoscaler watches traffic flow to the application, and scales the application up or down based on metrics.
+
+The default Autoscaler used in Knative deployments is the Knative Pod Autoscaler (KPA). However, Knative deployments can be configured to use the Kubernetes Horizontal Pod Autoscaler (HPA) instead.
+<!-- Question: removed mention of BYO autoscaler since we don't seem to doc this yet - does it need to be added, can we provide instructions for it yet?-->
+Knative Serving revisions are configured by default to use generic settings that are suitable for the majority of use cases. However, some workloads may required a custom, more finely-tuned configuration.
+This guide provides information about configuration options that you can modify to fit the requirements of your workload.
+
+## Autoscaling concepts
+
+The following sections describe important concepts and terms used when discussing autoscaling in Knative, as well as configuration details where applicable.
+
+* [Supported Autoscaler types](./supported-autoscaler-types.md) (includes configuration settings for enabling HPA)
+* [Global versus per revision settings](./global-vs-per-revision.md)
+* [KPA specific behavior](./kpa-specific.md)
```

Comment:
I would move this to the bottom of the list. I'd expect users to care about all the other bits first.